### DISCUSSION TITLE
3.0.0版本上传资源问题

### DISCUSSION THREAD

**Discussion by herman-8639** [apache/dolphinscheduler#13664] [Q&A]
1.postman调用 /dolphinscheduler/resources接口，可以正常上传文件（大文件小文件都可以）

2.管理界面上，使用admin用户上传文件，小文件正常上传，大文件（50M作用）无法上传
后台api日志，提示“分页查询资源列表错误”，该错误应该是上传接口默认回调，不会影响上传功能，没有更多日志

**Comment by Tandoy** [comment]
related: https://github.com/apache/dolphinscheduler/issues/10340   https://github.com/apache/dolphinscheduler/issues/10509
